torchbenchmark/models/tts_angular/__init__.py

```diff
@@ -0,0 +1,38 @@
+
+from ...util.model import BenchmarkModel
+from torchbenchmark.tasks import COMPUTER_VISION
+
+from .angular_tts_main import TTSModel
+
+class Model(BenchmarkModel):
+    task = COMPUTER_VISION.SEGMENTATION
+    def __init__(self, device=None, jit=False):
+        super().__init__()
+        self.device = device
+        self.jit = jit
+        self.model = TTSModel(device=self.device)
+
+    def get_module(self):
+        if self.jit:
+            raise NotImplementedError()
+        return self.model, []
+
+    def set_train(self):
+        # another model instance is used for training
+        # and the train mode is on by default
+        pass
+
+    def train(self, niterations=1):
+        # the training process is not patched to use scripted models
+        self.model.train()
```

Comment:
`niterations` is not used, we need to pass this parameter to train to control how many epochs we want to run for training and inference.